public/docs/content/getting_started/.pages.yml

```diff
@@ -0,0 +1,8 @@
+---
+  nav:
+    - Getting started: index.md
+    - Overview of core modules: core_modules.md
+    - Project structure: project_structure.md
+    - Creating a new construct: create_new_construct.md
+    - How to contribute: how_to_contribute.md
```

Comment:
This should be moved to the top.

Reply:
See comment above about moving it to a different menu item